app/experimenter/experiments/tests/test_forms.py

```diff
@@ -630,6 +630,26 @@ def test_form_checks_uniqueness_for_single_experiment_not_all(self):
         experiment2 = form2.save()
         self.assertEqual(experiment2.variants.count(), 3)
 
+    def test_form_valid_if_sizes_sum_to_100(self):
+        form = ExperimentVariantsAddonForm(
+            request=self.request, data=self.data
+        )
+        self.assertTrue(form.is_valid())
```

Comment:
So, I can understand why this test works. On the other hand, the data that makes this test valid and interesting is very far away from this test. I think it would be better to include the experiment sizes data directly in the test, even if it is redundant.

Reply:
This comment also applies to the other tests, which again rely on data that isn't clear and is far away from the test. I think that this hurts the readability of the tests, which makes it harder to maintain.